CONTRIBUTING.md

```diff
@@ -36,9 +36,15 @@ information on using pull requests.
 
 ### Style
 
-Our C++ code mostly follows [Google's C++ style guide][google-cpp-style]. We can
-take care of automatic code formatting for you when we import your pull
-requests.
+Our C++ code mostly follows [Google's C++ style guide][google-cpp-style].
+
+The github actions on a pull request will verify this style, so to make
```

Comment:
Negligible suggestion: you might want to use "Continuous Integration (CI)" instead of "github actions", so you/we don't need to update this in the future (as it's happening with many references to Travis in open source projects).